### PR TITLE
Layering change - belts and suits

### PR DESCRIPTION
**Pull request by McRamon**
## What Does This PR Do
Belt and Suit layers swapped, so now Suit (coats/jackets/armors etc) slot is layered above belts. lots of suits are wider than mob waist, while belt is "wrapped" around mob sprite therefore it looks bad on many of the suits, especially with high contrast between belt and suit (like pink coat and black security belt). 

## Why It's Good For The Game
Looks better that way. there are a few suits where it would (somewhat) make sense for them to go under belt logically (like some fullbody suits liek modsuit) but more often suit slot should really go above belt (all coats, jackets, labcoats, winter coats, etc etc etc), and with them belt wouldn't ruin the sprite and character look.

## Images of changes

image
image
## Testing
put on some suits and belts

## Changelog
:cl:
tweak: Belt sprite on mobs now goes under the suit (external wear liek jackets) slot so it doesnt ruin the suit sprites.
/:cl:

### CODE REVIEW

File: code/__DEFINES/misc_defines.dm [hunk 1]
```diff
@@ -182,8 +182,8 @@
 #define SHOES_LAYER				28
 #define GLOVES_LAYER			27
 #define EARS_LAYER				26
-#define SUIT_LAYER				25
-#define BELT_LAYER				24	//Possible make this an overlay of somethign required to wear a belt?
+#define BELT_LAYER				25	//Possible make this an overlay of somethign required to wear a belt?
```
Comment (Henri215) [inline code/__DEFINES/misc_defines.dm:185]:
```suggestion
#define BELT_LAYER				25	//Possible make this an overlay of something required to wear a belt?
```

**Comment by dj-34** [comment]
Finally

File: code/__DEFINES/misc_defines.dm [hunk 2]
```diff
@@ -182,8 +182,8 @@
 #define SHOES_LAYER				28
 #define GLOVES_LAYER			27
 #define EARS_LAYER				26
-#define SUIT_LAYER				25
-#define BELT_LAYER				24	//Possible make this an overlay of somethign required to wear a belt?
+#define BELT_LAYER				25	//Possible make this an overlay of something required to wear a belt?
```
Comment (FunnyMan3595) [inline code/__DEFINES/misc_defines.dm:185]: I think the comment here can go away, since we're actually using the layer.

**Review by Burzah — APPROVED** [review]

**Review by S34NW — APPROVED** [review]

**Comment by Spaghetti-bit** [comment]
Issue with this. Webbing and other belts go under the armor. A key note... the bandolier goes under armor.

**Pull request merged**